src/components/ui/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,302 @@
+    { title: 'Central Board', link: '/central-board' },
+    { title: 'FAQs', link: '/faqs' },
+  ];
+
+  const officesDropdownItems = [
+    { title: 'Office of the President', link: '/offices/president' },
+    { title: 'Office of the Vice President', link: '/offices/vice-president' },
+    {
+      title: 'Office of the Secretary General',
+      link: '/offices/secretary-general',
+    },
+    { title: 'Office of the Treasurer', link: '/offices/treasurer' },
+  ];
+
+  const infoPortalDropdownItems = [
+    { title: 'Bulletin', link: '/info-portal/bulletin' },
+    { title: 'GuideSite', link: '/info-portal/guidesite' },
+    { title: 'Event Finder', link: '/info-portal/event-finder' },
+  ];
+
+  const academixDropdownItems = [
+    { title: 'AcadHub', link: '/academix/acadhub' },
+    { title: 'AcadDrive', link: '/academix/acaddrive' },
+    { title: 'Atheneum', link: '/academix/atheneum' },
+  ];
+
+  const ArrowSize = 15;
+  const [openDropdowns, setOpenDropdowns] = useState<string[]>([]);
+  const [AcademixDropdowns, setAcademixDropdowns] = useState(false);
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
+
+  const toggleDropdown = (id: string, isDesktop: boolean) => {
+    setOpenDropdowns((prev) =>
+      isDesktop
+        ? prev.includes(id)
+          ? []
+          : [id]
+        : prev.includes(id)
+          ? prev.filter((drop) => drop !== id)
+          : [...prev, id]
+    );
+
+    if (id !== 'academix') setAcademixDropdowns(false);
+  };
+
+  const toggleMobileMenu = () => setIsMobileMenuOpen((prev) => !prev);
+
+  const handleKeyDown = (
+    event: React.KeyboardEvent<HTMLElement>,
+    action: () => void
+  ) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      action();
+    }
+  };
+
+  const DropdownItems = (items: { link: string, title: string; }[]) =>
+    items.map(({ title, link }) => (
+      <li
+        key={title}
+        className="cursor-pointer p-2"
+        tabIndex={0}
+        role="menuitem"
+      >
+        {/* for navigation routings */}
+        <Link href={link}>{title}</Link>
+      </li>
+    ));
+
+  return (
+    <nav className="bg-white py-3 px-6 shadow-[0_4px_20px_rgba(0,0,0,0.1)] rounded-[100px] flex justify-between items-center">
+      <div className="text-blue-900 font-bold text-xl flex items-center">
+        <Image src={logo} alt="Logo" width={90} height={90} />
+      </div>
+
+      {/* Toggle icons in mobile view */}
+      <div className="md:hidden">
+        <button
+          onClick={toggleMobileMenu}
+          className="text-blue"
+          aria-label="Toggle mobile menu "
+        >
+          {isMobileMenuOpen ? <IoClose size={40} /> : <HiBars3 size={40} />}
+        </button>
+      </div>
+
+      {/* Desktop menu */}
+      <ul
+        className="hidden md:flex space-x-6 text-blue font-semibold"
+        role="menu"
+      >
+        {/* Home and About Links */}
+        {navItems.slice(0, 2).map(({ title, link }) => (
```

Comment:
Separate the nav items into two separate constants instead of slicing. This will break the code if you add additional elements in the constant.